nengo/networks/ensemblearray.py

```diff
@@ -245,6 +245,14 @@ def add_output(self, name, function, synapse=None, **conn_kwargs):
             should stay as None, and synaptic filtering should be performed in
             the connection from the output node.
         """
+        if hasattr(self, name):
+            raise ValidationError(
+                "Cannot add output named %r; there is already an attribute by this name"
```

Comment:
Should this be added to the changelog either under fixed or changed? Since before it would "work" by overwriting it, and this could break someone's example if they were relying on this behaviour (or doing it accidentally)?